documentation/src/main/wiki/BestPractices.md

```diff
@@ -1,21 +1,28 @@
 ## Best practices and tips for writing AET test scripts
 
-#### Learn the basics of Selenium WebDriver and understand how to this knowledge when working with AET
-AET interacts with web pages through WebDriver so it's good to know how to use WebDriver. Knowing about it's limitations and what it requires from test scripts to work properly can help a lot when writing test scripts for AET. It's especially important to understand that in order to get stable tests AET scripts should include smartly located wait operations, just like WebDriver test scripts do.
+#### Learn the basics of Selenium WebDriver and understand how to use this knowledge when working with AET
+AET interacts with web pages through WebDriver so it's good to know how to use WebDriver. 
+Knowing about it's limitations and what it requires from test scripts to work properly can help a lot when 
```

Comment:
+Knowing its limitations and what it requires from test scripts to work properly can help a lot when